appcircle_ios_reactnative-codepush/1.1.0/component.yaml

```diff
@@ -0,0 +1,77 @@
+platform: common
+buildPlatform: ReactNative
+displayName: Appcircle CodePush
+description: "Release a React Native update to Appcircle CodePush."
+docsUrl: https://docs.appcircle.io
+requiredComponents: "appcircle_git_clone,appcircle_npm_yarn,appcircle_node_install"
+precedingComponents: 
+followingComponents: 
+webUrl: https://github.com/appcircleio/appcircle-codepush-component
+repoUrl: https://github.com/appcircleio/appcircle-codepush-component.git
+commit: e132aae
+inputs:
```

Comment:
My feedback for the inputs is the same as the feedback I gave in the repo below.

https://github.com/appcircleio/appcircle-codepush-component